src/service/Cinema.java

```diff
@@ -0,0 +1,58 @@
+package service;
+
+public class Cinema {
```

Comment:
This class is placed in the same package as CinemaService, you can use the default access modifier.

Reply:
Changed